integration_tests/src/main/python/data_gen.py

```diff
@@ -1031,3 +1031,7 @@ def append_unique_to_rows(rows):
     new_rows = append_unique_to_rows(collected)
     new_schema = StructType(existing_schema.fields + [StructField("uniq_int", IntegerType(), False)])
     return spark.createDataFrame(new_rows, new_schema)
+
+
+disable_field_id_write = {"spark.sql.parquet.fieldId.write.enabled": "false"}  # default is true
```

Comment:
Ideally the tests should not rely on defaults in Spark, as those defaults could change over time.  For example, if the test relies on fieldId write being enabled, it should explicitly set it rather than hope the default lines up with expectations.

Reply:
Done